docs/release/trg-0/trg-1-4.md

```diff
@@ -80,11 +80,23 @@ flowchart TD
     C -->|Three| F[fa:fa-car Car]
 ```
 
+### Draw.io Example
+
+This image was generated with [draw.io](https://app.diagrams.net/) + `version` and stored as `.svg` file within the repository.
```

Comment:
Is the used version really important here? Are there migration issues with saved draw.io exports? 
If so, I would add a small hint, how to find out, which version of draw.io is live, when using the public version.
What is missing from my point of view. is the reference to the draw.io export. AFAIK, you need to store the XML based `.drawio` file in order to import and edit it again